.github/workflows/linting.yml

```diff
@@ -0,0 +1,17 @@
+name: 'linting'
```

Comment:
Why did you split the test workflow into multiple workflows?

Reply:
They are conceptually different, so I'd like to see different status badges for each